App.tsx

```diff
@@ -106,10 +108,11 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                         description={'Not a single thing'}
                         IconClass={ChartLineVariant}
                         actions={
-                            <Button mode={'outlined'} labelStyle={{ color: PXBColors.blue[500] }}>
+                            <Button mode={'outlined'} icon={'plus'}>
```

Comment:
We should probably always be using the wrapper version of the Button. So if someone comes in and changes the variant on this to contained, it won't be broken.